src/components/mini-widgets/ModeSelector.vue

```diff
@@ -16,8 +16,14 @@ datalogger.registerUsage(DatalogVariable.mode)
 const vehicleStore = useMainVehicleStore()
 const currentMode = ref()
 
-watch(currentMode, () => {
+/* eslint-disable-next-line @typescript-eslint/no-unused-vars */
+watch(currentMode, (newVal, _oldVal) => {
   if (currentMode.value === undefined) return
+  // Fetch the current mode directly from the store to ensure it's different
+  if (newVal === vehicleStore.mode) {
+    console.log('mode is the same as the current mode, not commanding a mode change.')
```

Comment:
I think this can be a debug log. It would also be good to capitalize the first word in the sentence here.

"New mode is the same as the current one. No mode-change commands will be issued."

Reply:
fixed